roles/pulibrary.pulmap/meta/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+galaxy_info:
+  author: Princeton University Library
+  description: pulmap role
+
+  license: MIT
+
+  min_ansible_version: 2.8
+
+  platforms:
+    - name: Ubuntu
+      versions:
+        - 18.04
+dependencies:
+  - role: "pulibrary.samba"
+  - role: "pulibrary.ruby"
+  - role: "pulibrary.postgresql"
+  - role: "pulibrary.redis"
+  - role: "pulibrary.sidekiq_worker"
+  - role: "pulibrary.sneakers_worker"
+  - role: "pulibrary.blacklight-app"
+  - role: "pulibrary.rails-app"
+  - role: "pulibrary.mailcatcher"
```

Comment:
This is new. I don't think we need this for Pulmap.

Reply:
do you need to do tests for mail delivery on staging? If yes then you will need it.